docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -123,12 +123,12 @@ The probability of side effects is also taken from the literature, and if a pers
 BAU parameter data tables
 +++++++++++++++++++++++++
 
```

Comment:
Patients are initialized with GBD data on LDL-C., SBP, IKF (CKD state) and FPG (DM state), as well as sex, age, location.  

To initial simulants as "currently on Rx" (for those patients that already have an Rx at the start of the simulation), we need to transform the GBD estimates of LDL-C to remove the effect of current Rx.  The logic is this: the number of people with "low" LDL-C (e.g. LDL-C < 4.9 mmol/L) according to GBD includes those already on medication.  In order to identify these people and initialize them accordingly (initialize them with Rx), we use the following transformation:
Population with high LDL-C (adjusted from GBD) = (population with high LDL-C from GBD) / (1 - prob(Rx|high LDL-C)*prob(hitting target|Rx))
where prob(Rx|high LDL-C) comes from Table 2 and prob(hitting target|Rx) comes from Table XXX.

There are 3 ways a simulant is exposed to possible treatment:  background visits, post-MI visits (immediately after an MI), and follow-up visits (once they have been given Rx).  

GBD healthcare utilization is used to determine how often a simulant goes to the doctor for background visits; MI incidence determines how often a simulant goes to the doctor for a "post MI" visit, and "follow-up" visits are determined by the assumption on follow-up frequency (3-6 months for BAU and 4-6 weeks for the 2 intervention scenarios).

Reply:
Serena - I omitted the table for prob(hitting target|Rx) from the excel sheet I sent you.  I have emailed it to you, and it is highlighted in green.  I named it "Table XXX" here just so that I don't mess up your numbering scheme.  Please number it appropriately and change XXX.